src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBindingContext.cs

```diff
@@ -22,11 +23,18 @@ public class FunctionBindingContext
         /// <param name="functionInstanceId">The instance ID of the function being bound to.</param>
         /// <param name="functionCancellationToken">The <see cref="CancellationToken"/> to use.</param>
         /// <param name="trace">The trace writer.</param>
-        public FunctionBindingContext(Guid functionInstanceId, CancellationToken functionCancellationToken, TraceWriter trace)
+        /// <param name="functionDescriptor">Current function being executed. This will influence properties on the <see cref="SystemBindingData.MethodName"/>  </param>
+        public FunctionBindingContext(
+            Guid functionInstanceId, 
```

Comment:
This is a public type ( :( )  so I was being safe by providing the existing signature. Missing a FuncDescriptor just means we're missing 'sys', which is fine in the majority case. 

---
In reply to: [117081641](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r117081641) [](ancestors = 117081641)